postBuild

```diff
@@ -0,0 +1,4 @@
+set -ex
+
+# Add support for developing with git and GitHub
+conda install -c conda-forge jupyterlab-git gh-scoped-creds
```

Comment:
is `postBuild` run from within the conda environment? Or does this need to specify `-n notebook`?

Reply:
My understanding it is run within the already activated conda environment.